_config.yml

```diff
@@ -33,3 +34,8 @@ exclude:
   - cache
 collections:
   hackers:
+what_we_do: >
+  We are a self managed group that work all together and  believe into share
```

Comment:
We are a self managed group that work together and believe in sharing knowledge to improve and grow.